pyuvsim/tests/test_simsetup.py

```diff
@@ -452,13 +460,7 @@ def test_param_select_bls():
     uv_obj_bls2 = uv_obj_full.select(bls=[(40, 41), (42, 43), (44, 45)], inplace=False, metadata_only=True)
 
     uv_obj_bls.history, uv_obj_bls2.history = '', ''
-
-    nt.assert_equal(uv_obj_bls, uv_obj_bls2)
-
-    # check kwargs passed
-    param_dict['object_name'] = 'foo'
-    uv_obj_full, new_beam_list, new_beam_dict = pyuvsim.initialize_uvdata_from_params(param_dict)
-    nt.assert_equal(uv_obj_full.object_name, 'foo')
```

Comment:
Are  we sure we don't want these three lines?

Reply:
Hm I'm not sure why/how these got taken out. I'll take another look at them